Src/FluentAssertions/Formatting/FormattingOptions.cs

```diff
@@ -1,12 +1,19 @@
-﻿namespace FluentAssertions.Formatting;
+﻿using System.Collections.Generic;
+using FluentAssertions.Execution;
+
+namespace FluentAssertions.Formatting;
 
 public class FormattingOptions
 {
+    internal static readonly List<IValueFormatter> CustomFormatters = [];
```

Comment:
Do you mean that we should keep it in `Formatters`.. But I agree.. this was something I didn't get why, but because the API was approved as such, I moved this field